Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -32,4 +32,11 @@ void Processor::initialize(u32 cpu)
 
     g_current_processor = this;
 }
+
+[[noreturn]] void Processor::halt()
+{
+    for (;;) {
+        asm volatile("wfi");
+    }
+}
 }
```

Comment:
Please add a newline between the namespace curly braces

Reply:
Ah yep, will remind this when submitting my next PRs! Btw, maybe it is possible to enforce this using `clang-format`?

Reply:
Nope, unfortunately not